bytes/bytes/database/db_models.py

```diff
@@ -26,8 +26,10 @@ class RawFileInDB(SQL_BASE):  # type: ignore
     __tablename__ = "raw_file"
 
     id = Column(UUID, primary_key=True)
+
     secure_hash = Column(String(length=256), nullable=True)
     hash_retrieval_link = Column(String(length=2048), nullable=True)
+    signing_provider = Column(String(length=256), nullable=True)
```

Comment:
Maybe we should normalise this, i.e. create a signing provider table and make this a foreign key?

Reply:
Perhaps we should.. This would mainly save space given the append-only nature of these entities, at the cost of a join every time we fetch the meta data.

Reply:
My guess is the join would be very cheap to do because there are only a very limited number of signing providers. It will save space compared to storing the whole url in each row because the primary key of the table can just be a 32-bit int.

Reply:
As longs as we make it reasonably easy for a user to have the signing endpoint available next to the raw data for validation, all is well.